### DISCUSSION TITLE
how to buy Zotero Style Pro outside of China

### DISCUSSION THREAD

**Discussion by ulahcherubim** [MuiseDestiny/zotero-style#955] [Q&A] [answered]
Hello, I wanted to buy Zotero Style Pro, but couldn't figure out how. Can someone help me with the payment process, perhaps @MuiseDestiny?

**Comment by SthefanBerwanger** [comment]
It would be nice!

**Comment by lajustman** [comment]
Also interested in purchasing and not in China!

**Comment by jordsprakj** [comment, marked as answer]
Has anyone found a way to purchase it?

**Reply by MuiseDestiny** [reply]
<img width="1512" height="1475" alt="image" src="https://github.com/user-attachments/assets/4f35741e-abe0-4c10-9111-ae7fa3e2ac73" />

We recommend using Taobao (the second QR code). After placing your order, please specify Beijing, China as your address.

**Reply by jordsprakj** [reply]
Thank you

**Reply by nicolafern** [reply]
I have found that this doesn't work because Taobao prevents signing up without a Chinese phone number. Is there an alternative, such as a way to send money directly?

**Reply by jordsprakj** [reply]
> I have found that this doesn't work because Taobao prevents signing up without a Chinese phone number. Is there an alternative, such as a way to send money directly?

you have to make a taobao account on your phone so you can make an account with your email address. search on youtube how to use a taobao account. this worked for me and i bought it, so this does work.

**Comment by nicolafern** [comment]
I tried multiple times from laptop and phone. Closest I got was using the
Google login option from the app, then it wanted to send an auth code to my
email address, that came though, then it said my email address was invalid
to get an account. All the options using phone numbers said it wasn't
available for my UK number.

I tried several different ways, all of which were different in tiny ways.
The app being all in Chinese did not help but got through with translated
screen shots. Not sure what more I can do.

**Reply by jordsprakj** [reply]
don't use phone number only address. have you tried youtube. create account on phone with address. then login on website to change language to english. use a different address this time don't login via google. just use your email address

**Reply by nicolafern** [reply]
I finally managed to create an account and within seconds I'm now getting this message, so at minimum I can't do anything for three days. I have no idea what I did to trigger this, but I can't even use the search box now: 

Dear User, We have detected unusual activity on your account, which appears to involve the improper collection or use of the platform’s commercial information, including but not limited to the use of crawler tools, scripts, account renting, sharing, or selling of data. This poses a potential security risk. As a result, certain access functions of your account have been temporarily restricted and are expected to resume normal operation after 2026-01-28 18h. Please review your account and rectify any irregularities. Repeated violations may result in escalated actions. For more details, please visit the Security Center in the Taobao app (Account & Security > Security > Security Center).

**Reply by nicolafern** [reply]
I managed to get the account unblocked by talking to a support person and have now completed the purchase. It's taken me a good two hours across two days so it's not an easy process. I have successfully activated the plugins though so 🎉🙂

**Comment by MuiseDestiny** [comment]
Thank you very much for the support from users outside mainland China. In fact, the Pro version was developed specifically for Chinese users, and some features have not yet been internationalized in language. I will complete this in the future.

**Reply by nicolafern** [reply]
Thanks! It would be very helpful to have the remaining translations. In the meantime I'm making the most of Google Lens and Gemini to live translate where needed. I must say that I am very happy with the additional features. In fact, the value that is added by all of these plug-ins is immense. Thanks so much for the effort you have put in to enhancing Zotero!
